### DISCUSSION TITLE
how to log slack alert to console output  ?

### DISCUSSION THREAD

**Discussion by elvis-cai** [BerriAI/litellm#7371] [General]
as the title, seems we can only see it from slack channel

**Comment by elvis-cai** [comment]
cc @krrishdholakia @ishaan-jaff

**Reply by ghost** [reply]
Is there a specific slack alert you'd want to log on the console?

**Reply by elvis-cai** [reply]
we'd like to see all alert logs if possible @krrishdholakia
